common/flavorization/wc_title_holders.txt

```diff
@@ -86,128 +127,1878 @@ warlord_female = {
+	culture_groups = { dragon_group }
+}
+overlady_female_dragon {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	culture_groups = { dragon_group }
+}
+### Red Dragons
+dragonking_male_red_dragon {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+dragonqueen_female_red_dragon {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+dragonprince_male_red_dragon {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+dragonprincess_female_red_dragon {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+
+### Mantids ###
+emperor_male_mantid {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { mantid }
+}
+empress_female_mantid {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { mantid }
+}
+
+imperial_vizier_male_mantid = {
+	type = character
+	gender = male
+	special = councillor
+	tier = empire
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+imperial_vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = empire
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+grand_vizier_male_mantid = {
+	type = character
+	gender = male
+	special = councillor
+	tier = kingdom
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+grand_vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = kingdom
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+vizier_male_mantid = {
+	type = character
+	gender = male
+	special = councillor
+	tier = duchy
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = duchy
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+
+### Qiraji ###
+# Special
+emperor_ahn_qiraj_male_qiraji {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+	titles = { k_ahnqiraj }
+}
+empress_ahn_qiraj_female_qiraji {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+	titles = { k_ahnqiraj }
+}
+empire_ahn_qiraj = {
```

Comment:
This is VERY confusing. 2 titles with the same name.

Reply:
There are two emperors in Ahn'Qiraj in the lore, I did this for aesthetic purposes as obviously there cannot be two rulers for the imperial title.